README.md

```diff
@@ -23,11 +23,30 @@
 # Fingerprint Pro Cloudflare worker
 The Cloudflare worker is responsible for delivering the latest fingerprinting client-side logic and proxying identification requests and responses between your site and Fingerprint Pro's APIs.
 
+## Requirements
+
+### Usage Requirements
+
+Please see the [Setup Process](#setup-process) section
```

Comment:
The integration actually has a well defined usage requirements defined https://dev.fingerprint.com/docs/cloudflare-integration#setup 

You could briefly list them here and point to the guide for more details

Reply:
Updated usage requirements